packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.stories.ts

```diff
@@ -17,6 +17,13 @@ const createStory = subplatStoryWithProps(
   }
 );
 
+export const LayoutNoProduct = createStory(
+  {
+    reminderShortForm: true,
+  },
+  'Reminder short form - user has Firefox account, no specific product, shows Mozilla Privacy Policy and Firefox Cloud Terms of Service links, hides Terms and cancellation policy, Cancel/Reactivate subscription, and Update billing information links'
```

Comment:
I think it's OK if we want to leave this, but I think it might be harder to maintain/can easily become outdated and if we're going to be this specific, we should also be specific with the other subplat layout stories. You could also just shorten this to: `Reminder short form - no specified product` IMO to match the other stories.

Also, we were going to adjust the multiple product story at some point? I remember us bringing it up in a PR from last week or the week before but I can't remember what we said specifically (and I'm feeling lazy and not looking up my comment because it's after 6 😆)

Reply:
Updated reminder short form. 👍

Oh, I thought the `LayoutMultipleProducts` and its description of `Multiple products` sufficed. It was `LayoutNoProduct` before, which did not not make much sense. 👀